diff -urN old/etc/sysconfig/network-scripts/ifcfg-eth0 new/etc/sysconfig/network-scripts/ifcfg-eth0
--- old/etc/sysconfig/network-scripts/ifcfg-eth0	2014-10-13 12:27:47.071498975 +0530
+++ new/etc/sysconfig/network-scripts/ifcfg-eth0	2014-10-13 12:28:19.547660015 +0530
@@ -1,5 +1,5 @@
 DEVICE=eth0
-BOOTPROTO=none
+BOOTPROTO=dhcp
 ONBOOT=yes
 TYPE=Ethernet
 
diff -urN old/etc/sysconfig/network-scripts/ifcfg-eth1 new/etc/sysconfig/network-scripts/ifcfg-eth1
--- old/etc/sysconfig/network-scripts/ifcfg-eth1	2014-10-13 12:27:56.923547829 +0530
+++ new/etc/sysconfig/network-scripts/ifcfg-eth1	2014-10-13 12:28:30.419713927 +0530
@@ -1,4 +1,4 @@
 DEVICE=eth1
-BOOTPROTO=none
+BOOTPROTO=dhcp
 ONBOOT=yes
 TYPE=Ethernet
